docs/photoniq/est/manage-components.md

```diff
@@ -0,0 +1,144 @@
+  ![Components Tab](/img/photoniq/est/components-tab.png)
+- **REST API** - Macrometa provides API calls that allow you to view and manage components.
+
+## View Components
+
+<Tabs groupId="operating-systems">
+<TabItem value="console" label="Web Console">
+
+Follow these instructions to view components and component settings in the web console.
+
+1. Log in to your Edge Side Tagging dashboard.
+2. Click **Components**.
+
+   Macrometa displays a list of all components.
+
+3. To view component settings, click the stacked dots on the line of the component that you want to view settings for, and then click **Edit**.
+
+   Macrometa displays component settings.
+
+</TabItem>
+<TabItem value="api" label="REST API">
+
+Use our interactive API Reference with code generation in 18 programming languages to:
+
+- [Get all component settings](https://www.macrometa.com/docs/apiEst#/paths/api-est-v1-components-settings/get)
+- [Get component settings for a specific component](https://www.macrometa.com/docs/apiEst#/paths/api-est-v1-components-settings-component/get)
+
+</TabItem>
+</Tabs>
+
+### Fields
+
+- **Name** - Name the component was given when created.
+- **Type** - Type of component, such as Bing or Reddit.
+- **Activity Last 24 Hours** - Displays activity from the last 24 hours.
+- **Status** - Indicates whether the component is active or not. If it is on, then the component is enabled.
+
+## Enable or Disable Components
+
+<Tabs groupId="operating-systems">
+<TabItem value="console" label="Web Console">
+
+Follow these instructions to enable or disable components in the web console.
+
+1. Log in to your Edge Side Tagging dashboard.
+2. Click **Components**.
+3. In the **Status** column, click the toggle to enable or disable the component.
+
+</TabItem>
+<TabItem value="api" label="REST API">
+
+Use our interactive API Reference with code generation in 18 programming languages to:
+
+REVIEWERS - I have no idea which API call I would use for this task based on the API Reference. There is no information.
+
+</TabItem>
+</Tabs>
+
+## Add New Components
+
+<Tabs groupId="operating-systems">
+<TabItem value="console" label="Web Console">
+
+Follow these instructions to add a new component in the web console.
+
+1. Log in to your Edge Side Tagging dashboard.
+2. Click **Components**.
+3. Click **New Component**.
+4. Click the component that you want to add.
+5. Enter a **Component Name** and then click **Continue**.
+6. Enable permissions and then click **Continue**. Each component requires different permissions, which are described in the console.
+7. Enter any required setup information and then click **Continue**. Each component has different setup fields, which are described in the console.
+
+</TabItem>
+<TabItem value="api" label="REST API">
+
+Use our interactive API Reference with code generation in 18 programming languages to:
+
+REVIEWERS - Do users need to get the component metadata before they add a new component so they know what to enter?
```

Comment:
Yes, when user clicks on new component 
GET API api/est/v1/components/supported-components
Is called listing summary of all the supported components

When user selects specific component to configure , then 
GET API /api/est/v1/components/supported-components/{component}
Is called , which makes all related metadata available to the user

Reply:
I'm not talking about the UI. If I was just using the API, what would I need to do in order to add a new component? I am guessing get the specific supported component to know what parameters to enter, then enter those values in the add component endpoint?

Reply:
User would need to call GET API /api/est/v1/components/supported-components/{component}. to get available settings for specific component